src/AnchorsList.json

```diff
@@ -0,0 +1,32 @@
+{

```

Comment:
You don't need to create object with only one field. You can create JSON with array as top element.